cypress/e2e/EngageSphere_GUI.cy.js

```diff
@@ -0,0 +1,57 @@
+// Renderiza o cabeçalho com um h1 e alternância de tema
+describe('Teste do cabeçalho e alternância de tema', () => {
+    beforeEach(() => {
+      cy.visit('/');
+    });
+
+    it('Deve renderizar o cabeçalho com um h1', () => {
+      cy.contains('h1', 'EngageSphere').should('be.visible');
+    });
+
+    it('Deve alternar o tema', () => {
+      cy.get('#theme-toggle-button').should('exist');
+      cy.get('body').should('have.attr', 'data-theme', 'light');
+      cy.get('#theme-toggle-button').click();
+      cy.get('body').should('have.attr', 'data-theme', 'dark');
+    });
+  });
+
+//Mostra a saudação padrão (ou seja, Hi there! ...)
+describe('Teste de saudação', () => {
+    it('Deve mostrar a saudação padrão "Hi there!"', () => {
+      cy.visit('http://localhost:3000/');
```

Comment:
Visto que a `baseUrl` está definida no arquivo de configurações do Cypress (`cypress.config.js`), não há necessidade de passar tal informação aqui. A URL relativa já basta.

```suggestion
      cy.visit('/');
```